examples/darkmode/src/components/ThemeSelect.astro

```diff
@@ -0,0 +1,28 @@
+<theme-selector></theme-selector>
+<script is:inline>
+  if (!customElements.get("theme-selector")) {
+    customElements.define(
+      "theme-selector",
+      class extends HTMLElement {
+        connectedCallback() {
+          this.innerHTML = `
+            <select onchange="theme.setTheme(this.value)" aria-label="Select Theme">
```

Comment:
So that's the trick to get the selected option? Not fan of using a string for it, is there at least a way to get syntax highlighting with a special comment?

Reply:
agreed, I updated this